FHEM/00_SIGNALduino.pm

```diff
@@ -557,8 +557,18 @@ sub SIGNALduino_Set($$@) {
   
   if($cmd eq "raw") {
     $hash->{logMethod}->($name, 4, "$name: Set, $cmd $arg");
-    #SIGNALduino_SimpleWrite($hash, $arg);
-    SIGNALduino_AddSendQueue($hash,$arg);
+		if ($arg =~ m/^Wseq /) {
+			my @args = split(' ', $arg);
+			foreach my $argcmd (@args) {
+				if ($argcmd ne "Wseq") {
+					#Log3 $name, 4, "set $name raw Wseq: $argcmd";
+					SIGNALduino_AddSendQueue($hash,$argcmd);
```

Comment:
Ich glaube ich könnt hier so ziemlich jedes Kommando in der Sequenz mit übergeben. Also nicht nur W... Kommandos.

Reply:
Ich habe den Vorschlag übernommen von @Ralf9 damit wir einen PR haben. Wenn man dies optimieren kann, dann gern sind wir offen für diese Erleichterung ;-)

Reply:
> @HomeAutoUser
> 
> Ich habe nichts dagegen, dass mehrere Register auf einen Rutsch verändert werden können.
> Ich bin nur nicht überzeugt, dass wir dem raw Kommandos,was Befehle an die Firmware übergeben soll Logik einbauen.
> 
> Folgendes, nicht getestet müsste bereits heute funktionieren
> `set sduino raw W067A#W070E`
> Einziger Haken an der Sache. Ich habe den Eingangspuffe auf 14 Zeichen begrenzt. 13 Zeichen darf der Befehl aktuell nicht überschreiten :(
Ich konkretisiere. Ein einzelner Befehl in der Kette darf diese Länge nicht überschreiten und das sollte bei den Registern ja nie der Fall sein :)